basics/gradle.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: content
+title: Gradle Integration
+author: wolfram
+as_version: 3.0
+cl_version: 1.0.2 or 1.1.0-beta5
+order: 6
+---
+
+### Top level build configuration
+You can add ConstraintLayout as you do with other dependencies: By adding it to your gradle `build.xml` file.
```

Comment:
`By` shouldn't have a capital `B`, it doesn't follow a full stop.

Reply:
Didn't know that. In Germany it's the other way round: Always use capital letters after colons. Will change that.

Reply:
Yeah some languages have "non-standard" rules when it comes to typography. I knew for example French puts a space before some punctuation (e.g., question marks and exclamation marks) but in English you generally don't do that